From 6cfffc107a070c8602a6512d426cd5ec28614d90 Mon Sep 17 00:00:00 2001
From: Student Main <studentmain@aosc.io>
Date: Tue, 6 Aug 2024 17:19:57 +0000
Subject: [PATCH] edit systemd service file to met need of AOSC OS

uncommented database service dependency
change user and group name to 'forgejo'
---
 contrib/systemd/forgejo.service | 31 ++++++++++++++-----------------
 1 file changed, 14 insertions(+), 17 deletions(-)

diff --git a/contrib/systemd/forgejo.service b/contrib/systemd/forgejo.service
index ee019e11ea78..6fa5b497e1eb 100644
--- a/contrib/systemd/forgejo.service
+++ b/contrib/systemd/forgejo.service
@@ -5,22 +5,19 @@ After=network.target
 ###
 # Don't forget to add the database service dependencies
 ###
-#
-#Wants=mysql.service
-#After=mysql.service
-#
-#Wants=mariadb.service
-#After=mariadb.service
-#
-#Wants=postgresql.service
+
+Wants=mysqld.service
+#After=mysqld.service
+
+Wants=postgresql.service
 #After=postgresql.service
-#
-#Wants=memcached.service
+
+Wants=memcached.service
 #After=memcached.service
-#
-#Wants=redis.service
+
+Wants=redis.service
 #After=redis.service
-#
+
 ###
 # If using socket activation for main http/s
 ###
@@ -53,15 +50,15 @@ After=network.target
 # LimitNOFILE=524288:524288
 RestartSec=2s
 Type=simple
-User=git
-Group=git
+User=forgejo
+Group=forgejo
 WorkingDirectory=/var/lib/forgejo/
 # If using Unix socket: tells systemd to create the /run/forgejo folder, which will contain the forgejo.sock file
 # (manually creating /run/forgejo doesn't work, because it would not persist across reboots)
 #RuntimeDirectory=forgejo
-ExecStart=/usr/local/bin/forgejo web --config /etc/forgejo/app.ini
+ExecStart=/usr/bin/forgejo web --config /etc/forgejo/app.ini
 Restart=always
-Environment=USER=git HOME=/home/git FORGEJO_WORK_DIR=/var/lib/forgejo
+Environment=USER=forgejo HOME=/home/forgejo FORGEJO_WORK_DIR=/var/lib/forgejo
 # If you install Git to directory prefix other than default PATH (which happens
 # for example if you install other versions of Git side-to-side with
 # distribution version), uncomment below line and add that prefix to PATH

base-commit: 12a277ed65220cc152419242280f426e84ba0e61
-- 
2.47.0

